### DISCUSSION TITLE
Training with forces doesn't minimize loss

### DISCUSSION THREAD

**Discussion by pablo-unzueta** [mir-group/nequip#353] [Q&A]
Hi Everyone,

I have a dataset that I've quadruple checked. If I do energy+force training, the validation loss practically stays constant. However, If I train only with only energies, the validation loss decreases as the training progresses. Any Ideas? I know this might not be an Allegro/Nequip specific problem, but I thought I would ask here. I will also try with other MLFFs soon.

Thanks!
Pablo

**Comment by Linux-cpp-lisp** [comment]
Hi Pablo,

Can you provide any more details about the dataset, perhaps by email (see my profile) if that works better for you?

Have you tried force only training? If force only or energy only training both work but joint doesn't, then almost certainly the two are inconsistent (different units? the whole VASP "free energy" vs "energy" data labeling thing?)
